v3/src/models/document/base-document-content.ts

```diff
@@ -146,6 +148,14 @@ export const BaseDocumentContentModel = types
     },
     setTileDimensions(tileId: string, dimensions: { width?: number, height?: number }) {
       self.findRowContainingTile(tileId)?.setTileDimensions(tileId, dimensions)
+    },
+    broadcastMessage(message: DIMessage, callback: iframePhone.ListenerCallback) {
+      const tileIds = self.tileMap.keys()
+      if (tileIds) {
+        Array.from(tileIds).forEach(tileId => {
+          self.tileMap.get(tileId)?.content.broadcastMessage(message, callback)
+        })
+      }
     }
```

Comment:
I would move this up to `document-content.ts` where more client-specific behavior lives.